pom.xml

```
@@ -54,15 +54,16 @@ THE SOFTWARE.
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/jenkins-clone-workspace-scm-plugin</gitHubRepo>
-    <jenkins.version>2.346.3</jenkins.version>
+    <jenkins.version>2.361.4</jenkins.version>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.346.x</artifactId>
-        <version>1763.v092b_8980a_f5e</version>
+        <!-- <artifactId>bom-2.346.x</artifactId> -->
+        <artifactId>bom-2.361.x</artifactId>
+        <version>1723.vcb_9fee52c9fc</version>
```

Newer version is better than older version.  See the latest released version in the [bom releases](https://github.com/jenkinsci/bom/releases).
```suggestion
        <version>1763.v092b_8980a_f5e</version>
```